frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -563,6 +563,12 @@ search-TimeRangeFilter--edit-range = <glyph></glyph>Edit Range
 search-TimeRangeFilter--save-range = Save Range
 
 
+## User Avatar
+## Shows user Avatar with alt text
+
+user-avatar-anonymous = Anonymous User
+user-avatar = User Profile
```

Comment:
These do not respect our [naming convention for l10n IDs](https://github.com/mozilla/pontoon/tree/master/frontend#adding-localized-content).